scripts/variantstore/wdl/GvsUtils.wdl

```diff
@@ -1154,10 +1167,11 @@ task SelectVariants {
     runtime {
         docker: gatk_docker
         cpu: 1
-        memory: "${memory_mb} MiB"
+        memory: memory_gib + " GB"
```

Comment:
this may be silly, but I don't know why you are changing how we're doing the string interpolation and setting the memory field here.  Was the old way broken?  Or is this just to bring it in line with how we do things elsewhere?

Reply:
I actually just lifted this from the other example of retry with more memory (in Extract) - really didn't think much of the change - but agree that we should stick with how we do things.